pages/register.js

```diff
@@ -0,0 +1,299 @@
+import React, { useState, useContext, useEffect, useRef } from 'react';
+import { isEmail } from "validator";
+import { ThemeContext } from "styled-components";
+import PageTemplate from '../frontend/components/templates/PageTemplate';
+import { SystemComponent, SystemSpan } from '../frontend/components/atoms/SystemComponents';
+import Subtitle from "../frontend/components/atoms/Subtitle";
+import Card from '../frontend/components/atoms/Card';
+import FieldSection from "../frontend/components/molecules/Form/FieldSection";
+import SubmitButton from '../frontend/components/atoms/SubmitButton';
+
+const TERMDESCRIPTIONS = [
+    'Academic term, active on Waterloop in-person', 
+    'Academic term, active on Waterloop remotely', 
+    'Co-op term, working on Waterloop remotely', 
+    'Co-op term, active on Waterloop in-person', 
+    'Not active on Waterloop this term', 
+    'Other',
+];
+
+const PREVTERMS = ['F21', 'S21', 'W21', 'F20', "S20",'W20',];
+
+const FUTURETERMS = ['W22', 'S22', 'F22', 'W23', "S23",'F23',];
+
```

Comment:
Would PREVTERMS and FUTURETERMS have to be changed manually every single term?

Reply:
Ideally we would like to automatically update these options. I think for the MVP, we can leave it this way.

Reply:
sounds good!